src/controllers/messages/users/buyer.js

```diff
@@ -119,8 +126,9 @@ async function addUserToQueue(recipient, listingId) {
   }
 
   await Promise.all(updates);
+  promptNextAction(recipient, listingId);
```

Comment:
Should you `await` this call?

Reply:
Don't have to, since the next two sends are to the seller not the buyer.